views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,4 @@
+<a href='/beers/{{id}}'></a><h1> {{name}}</h1>
+<img src='{{image_url}}' alt='beer' />
+<p> {{description}} </p>
+<p> {{tagline}} </p>
```

Comment:
Here the link that you are giving to the `<a>` tag is `/beers/{{id}}` but the path that you specified in the routes id `/beer/{{id}}` also you're `<a>` is empty so the link will not work, to make it work you wrap it around all the tags so in this way no meter where you click you will go to the beer details page:

```
<a href='/beer/{{id}}'>

  <h1> {{name}}</h1>
  <img src='{{image_url}}' alt='beer' />
  <p> {{description}} </p>
  <p> {{tagline}} </p>

</a>
```